.github/workflows/deploy-dev.yml

```
@@ -208,6 +238,7 @@ jobs:
       image_tag: dev-${{ github.sha }}
       commit_sha: ${{ github.sha }}
       deploy_actor: ${{ github.actor }}
+      commit_sha: ${{ github.sha }}
```

actionlint (1.7.12)

[error] key "commit_sha" is duplicated in "with" section. previously defined at line:239,col:7. note that this key is case insensitive

(syntax-check)